cfgov/ask_cfpb/scripts/export_ask_data.py

```diff
@@ -0,0 +1,143 @@
+from __future__ import unicode_literals
+
+import datetime
+
+import HTMLParser
+from django.utils import html
+
+from paying_for_college.csvkit import csvkit
+
+from ask_cfpb.models import Answer
+
+html_parser = HTMLParser.HTMLParser()
+
+HEADINGS = [
+    'ASK_ID',
+    'Question',
+    'ShortAnswer',
+    'Answer',
+    'URL',
+    'SpanishURL',
+    'Topic',
+    'SubCategories',
+    'Audiences',
+    'RelatedQuestions',
+    'RelatedResources',
+]
+
+SPANISH_HEADINGS = [
+    'ASK_ID'.encode('UTF16'),
+    'SpanishURL'.encode('UTF16'),
+    'SpanishAnswer'.encode('UTF16'),
+]
+
+
+def clean_and_strip(data):
+    unescaped = html_parser.unescape(data)
+    return html.strip_tags(unescaped).strip()
+
+
+def clean_and_strip_spanish(data):
+    unescaped = html_parser.unescape(data)
+    return html.strip_tags(
+        unescaped).strip().replace(
+        '\t', ' ').replace(
+        '\n', ' ').replace(
+        '\r', ' ')
```

Comment:
the UTF16 encoding was creating massive delimiting errors if I didn't strip out line endings and tabs. This doesn't happen with UTF8.